code/modules/organs/internal/internal_organ_processes.dm

```diff
@@ -148,6 +148,7 @@
 
 	if(blood_volume < total_blood_req)
```

Comment:
Does this not scale with how much oxygen organs use? So someone with modded organs would die from less bloodloss

Reply:
Okay, I codedived. So what this means is that if blood is below blood required + 20, you receive 2 oxyloss each tick. But if your blood is below required, you receive 4.5 oxyloss?

Is there math behind why this number is so specific? If there is, can you include it as a comment?